react/src/components/FileBrowserModal/FileBrowserModal.tsx

```diff
@@ -14,31 +17,62 @@ const { Text } = Typography;
 const FileBrowserModal = ({
   isOpen,
   toggle: parentToggle,
+  onImported,
   allowedFileExtensions = [],
 }: FileBrowserModalProps) => {
+  const [selectedFiles, setSelectedFiles] = useState<File[]>([]);
+
   const handleClose = () => {
     parentToggle();
   };
 
+  const handleFileSelect = (files: File[]) => {
+    console.log(files);
+    setSelectedFiles(files);
+  };
+
+  const handleImport = () => {
+    console.log(selectedFiles);
+    if (onImported) {
+      const tapisFilePaths = convertFilesToTapisPaths(selectedFiles);
+      onImported(tapisFilePaths);
+    }
+    handleClose();
+  };
+
   return (
     <Modal
       title={<Header style={{ fontSize: '2rem' }}>Select Files</Header>}
       open={isOpen}
       onCancel={handleClose}
+      zIndex={
```

Comment:
The zIndex can be set in the theme src/index.tsx like this:
```
Modal: {
      zIndexBase: 2000,
},
```
Here's the antd documentation [https://ant.design/docs/react/customize-theme#seedtoken](https://ant.design/docs/react/customize-theme#seedtoken)

Reply:
awesome. put in https://github.com/TACC-Cloud/hazmapper/pull/315